organization/mission.md

```diff
@@ -14,16 +14,11 @@ To enable public-purpose software and policy that is open and collaborative.
 
 * We are a non-profit and non-partisan organization.
 * We publish under licenses approved by the [Open Source Initiative](https://opensource.org/licenses) or [Creative Commons](https://creativecommons.org/licenses/) licenses that do not limit commercial reuse.
-* We are controlled by [our members](members.md), who are wholly publicly controlled organizations.*
-* We strive to keep our membership as accessible as possible to potential members, regardless of size or location.
 * We are committed to working with other organizations (commercial or non-profit) to achieve our mission.
-* We prioritize long term sustainability and growth for the benefit of our members.
```

Comment:
Are we skipping the priority completely, or is it just who it is for that is different? A lighter change would have been to just change to "We prioritize long term sustainability."

Reply:
Made a tweak - what do you think?